From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 4 May 2020 11:06:04 +0100
Subject: configs: Add missing PPS configs

The arm64 bcm2711_defconfig was inexplicably missing three PPS config
options. Add them, along with another missing option from
bcmrpi3_defconfig.

See: https://github.com/raspberrypi/linux/issues/3593

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/configs/bcm2711_defconfig | 3 +++
 arch/arm64/configs/bcmrpi3_defconfig | 1 +
 2 files changed, 4 insertions(+)

diff --git a/arch/arm64/configs/bcm2711_defconfig b/arch/arm64/configs/bcm2711_defconfig
index 337bb22a2714..e8b584fa28ea 100644
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -669,6 +669,9 @@ CONFIG_SPI_BCM2835AUX=m
 CONFIG_SPI_GPIO=m
 CONFIG_SPI_SPIDEV=m
 CONFIG_SPI_SLAVE=y
+CONFIG_PPS=m
+CONFIG_PPS_CLIENT_LDISC=m
+CONFIG_PPS_CLIENT_GPIO=m
 CONFIG_PINCTRL_MCP23S08=m
 CONFIG_GPIO_SYSFS=y
 CONFIG_GPIO_BCM_VIRT=y
diff --git a/arch/arm64/configs/bcmrpi3_defconfig b/arch/arm64/configs/bcmrpi3_defconfig
index 6a698ba0a539..c6613bbb8286 100644
--- a/arch/arm64/configs/bcmrpi3_defconfig
+++ b/arch/arm64/configs/bcmrpi3_defconfig
@@ -609,6 +609,7 @@ CONFIG_GPIO_BCM_VIRT=y
 CONFIG_GPIO_PCA953X=m
 CONFIG_GPIO_ARIZONA=m
 CONFIG_GPIO_STMPE=y
+CONFIG_GPIO_MOCKUP=m
 CONFIG_W1=m
 CONFIG_W1_MASTER_DS2490=m
 CONFIG_W1_MASTER_DS2482=m
